src/test/java/duke/TaskListTest.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * TaskListTest class to test TaskList object methods.
+ */
+public class TaskListTest {
+
+    /**
+     * Tests for adding a Task object to TaskList.
+     */
+    @Test
+    public void addTaskTaskAdded() {

```

Comment:
For your test cases (in other documents as well), perhaps you could follow this naming format indicated in the coding standards:

featureUnderTest_testScenario_expectedBehavior()
```suggestion
    public void addTask_TaskAdded() {

```

Reply:
Thanks for the suggestion!